BTC/implementation-of-the-bitcoin-system.md

```diff
@@ -0,0 +1,211 @@
+# L5 Implementation of the Bitcoin System
```

Comment:
Final Decision: The title should be like P5 not L5.

Reply:
done.